test/jobs/inventory_scheduled_sync_test.rb

```diff
@@ -5,6 +5,7 @@ class InventoryScheduledSyncTest < ActiveSupport::TestCase
   include ForemanTasks::TestHelpers::WithInThreadExecutor
 
   test 'Schedules an execution if auto upload is enabled' do
+    Setting[:use_local_advisor_engine] = false
```

Comment:
`use_local_advisor_engine` is a SETTING (set by installer options and/or settings.yaml), not a Setting (set by users via api/UI/Hammer). All of these `Setting[]` calls should not be needed. Instead the tests should probably mock `ForemanRhCloud.with_local_advisor_engine?` and make it return as needed.